lib/Devel/hdb/App.pm

```diff
@@ -151,6 +152,9 @@ sub init_debugger {
     local $@;
     eval { $self->load_settings_from_file() };
 
+    system { $Devel::hdb::LAUNCH } $Devel::hdb::LAUNCH, $self->_gui_url()
```

Comment:
If this ```system()``` fails to start the requested program, it should print an error including ```$!``` so the user can figure out what went wrong.

Reply:
Also a good thought, but I am not sure that $! is the correct variable. $! is Perl's representation of the C errnum/errstr stuff. I think things like IPC::System::Simple throw an exception if the system() fails. But I will try to come up with something.

Reply:
If you use ```$!``` in string context, it'll stringify errno.  For the easy case where you specify a program that doesn't exist, then the ```system()``` will fail.  ```$?``` is -1 and ```$!``` is "No such file or directory".  Other errors like the internal fork() failing will stringify similarly.